packages/protocol/genesis/test_config.js

```diff
@@ -111,7 +111,7 @@ module.exports = {
     };
   },
   param1559: {
-    gasExcess: 1,
+    gasExcess: 20000000000,
```

Comment:
hmmm but gasExcess can go down to `1`, so when that happens it will take much longer for it to reach this much higher value again no? If so we probably do want to enforce a gasExcess minimum to whatever results in a 1 wei basefee.

Reply:
I also think it's a good idea to apply a min value for gasExcess.

Reply:
I guess at that point maybe https://github.com/taikoxyz/taiko-mono/pull/16551 is indeed just simpler if we also confirmed that works.

Reply:
See https://github.com/taikoxyz/taiko-mono/pull/16555 please

Reply:
If possible, please approve this PR to merge it first to main.